support/index.html

```diff
@@ -0,0 +1,473 @@
+<!DOCTYPE html>
```

Comment:
@dayleparker Is a change to this file supposed to be in this PR?

Reply:
I am not sure what happened here. I'm starting again in a new PR :/ I will delete this one.